deploy/pms/install.sh

```
@@ -24,26 +24,26 @@ function installing_pms() {
   $COPY_UTIL configmap config-server-share config-server $NS
 
   INTERNAL_API_HOST=$(kubectl get cm global -o jsonpath={.data.mosip-api-internal-host})
-  PMP_UI_HOST=$(kubectl get cm global -o jsonpath={.data.mosip-pmp-ui-host})
+  PMP_UI_HOST=$(kubectl get cm global -o jsonpath={.data.mosip-pmp-host})
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)